pubspec.yaml

```diff
@@ -35,6 +35,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  http: ^1.1.0
```

Comment:
why do you have the http library added? are you using it?

Reply:
no, i've never used it, i added it at the beginning of the project because a misunderstanding and i completely forgot to delete it